### PR TITLE
[IN PROGRESS] Added Custom Wall

### PR DESCRIPTION
**Pull request by shreeshjha**
- [x] Read and understood [see CONTRIBUTING.md](https://github.com/fossasia/labyrinth/blob/master/CONTRIBUTING.md)
- [x] Included a preview link and screenshot showing after and before the changes.
- [x] Included a description of change below.
- [x] Tried to [squash the commits][squash].
- [x] Added Screenshot / Path to tile to Indicate Location of Change in Game

### Changes done in this Pull Request

- View my work at https://pr-996-fossasia-labyrinth.surge.sh

- Fixes #<Add issue number here>

### The problem I want to solve or the facility I want to improve is/are

Add a new CustomRoom tile that is majestic.

### My steps to solve it

  Made a wall, door, and floor in Inkscape and added them to the tiles folder
Changed tiles.js to add a new tile
Changed a currently unused/unavailable tile in levels.js to replace with the CustomRoom tile (Replaced tile was inaccessible and was a duplicate)

### Screenshots, Location of Changes, Path to Tile

screenshot from 2018-12-06 22-43-09

[squash]: https://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

### CODE REVIEW

**Comment by shreeshjha** [comment]
Sorry Guys I am unable to check how my wall looks in the game that's why I didn't provided the screenshot.

File: js/levels.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ function createFirstLevel()
     [door.handy, door.boat, door.river, desert.universe, door.cake, door.newYear],
     [door.both, door.square, door.quake, desert.universe, door.cake, door.mountains],
     [door.both, door.boat, door.river, desert.universe, door.oldRoom, door.mountains],  
+    [door.both, door.boat, door.river, desert.universe, door.oldRoom, door.CustomRoom],
```
Comment (abishekvashok) [inline js/levels.js]: That won't work

Comment (shreeshjha) [reply js/levels.js]: So what should I do in this line

Comment (abishekvashok) [reply js/levels.js]: Instead of adding a line, delete one wall used more than once and add yours.

Comment (shreeshjha) [reply js/levels.js]: Sure thanks

**Comment by YashKumarVerma** [comment]
Change the PR title when you want it to be reviewed.

**Comment by shreeshjha** [comment]
Sure

File: tiles/rooms/wall/CustomWall.svg [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+           rdf:resource="http://purl.org/dc/dcmitype/StillImage" />
+        <dc:title></dc:title>
+      </cc:Work>
+    </rdf:RDF>
+  </metadata>
+  <g
+     inkscape:label="Layer 1"
+     inkscape:groupmode="layer"
+     id="layer1">
+<<<<<<< HEAD
```
Comment (YashKumarVerma) [inline tiles/rooms/wall/CustomWall.svg]: NEEDED ?

File: tiles/rooms/wall/CustomWall.svg [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+           id="path178"
+           inkscape:connector-curvature="0" />
+        <path
+           style="fill:#ff6600;fill-opacity:1;stroke-width:0.20851807"
+           d="m 118.68533,151.27626 c 0,-8.73826 0.0126,-8.8795 0.84006,-9.45911 0.46203,-0.32361 0.9723,-0.58839 1.13393,-0.58839 0.16163,0 0.29387,3.99273 0.29387,8.87273 v 8.87273 l -1.13393,0.58638 -1.13393,0.58638 z"
+           id="path180"
+           inkscape:connector-curvature="0" />
+      </g>
+    </g>
+=======
```
Comment (YashKumarVerma) [inline tiles/rooms/wall/CustomWall.svg]: required ?

File: tiles/rooms/wall/CustomWall.svg [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+       id="path178"
+       inkscape:connector-curvature="0" />
+    <path
+       style="fill:#ff6600;fill-opacity:1;stroke-width:0.37797621"
+       d="m 118.68533,151.27626 c 0,-8.73826 0.0126,-8.8795 0.84006,-9.45911 0.46203,-0.32361 0.9723,-0.58839 1.13393,-0.58839 0.16163,0 0.29387,3.99273 0.29387,8.87273 v 8.87273 l -1.13393,0.58638 -1.13393,0.58638 z"
+       id="path180"
+       inkscape:connector-curvature="0" />
+>>>>>>> 7e431975b9bd59b297570620e3c62a4f44f35e7e
```
Comment (YashKumarVerma) [inline tiles/rooms/wall/CustomWall.svg]: required ?

**Pull request closed without merging**